core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -407,6 +408,8 @@ public class DefaultFileSystemMaster extends CoreMaster
   /** Used to check pending/running backup from RPCs. */
   private final CallTracker mStateLockCallTracker;
 
+  final Clock mClock;
```

Comment:
I believe this is dependency injection so we can use a manually controlled clock in the UT

Reply:
Yes it allows for reliable testing. I believe it is also the preferred way to use a clock since java 8. https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html